storage/store.go

```diff
@@ -1007,8 +1045,19 @@ func (s *Store) GroupStorage(groupID uint64) multiraft.WriteableGroupStorage {
 	defer s.mu.RUnlock()
 	r, ok := s.ranges[int64(groupID)]
 	if !ok {
-		log.Warningf("%p requested nonexistent range with raft ID %d", s, groupID)
-		return nil
+		var err error
+		r, err = NewRange(&proto.RangeDescriptor{
+			RaftID: int64(groupID),
```

Comment:
Asked @spencerkimball the same thing a while ago, looked like we should be able to homogenize this but just haven't gotten around to it yet.